src/app/events/services/test-state.service.ts

```diff
@@ -1,12 +1,13 @@
 import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
+import { BehaviorSubject, Observable } from 'rxjs';
 import { Course } from 'src/app/shared/models/course.model';
 import { CoursesRestService } from 'src/app/shared/services/courses-rest.service';
 import { LoadingService } from 'src/app/shared/services/loading-service';
 
 @Injectable()
 export class TestStateService {
   loading$ = this.loadingService.loading$;
+  search$ = new BehaviorSubject<string>('');
 
   // course$ = this.courseId$.pipe(switchMap((id) => this.state.getCourse(id)));
```

Comment:
Braucht es das noch - könnte man das auch löschen?

Reply:
Ja, das braucht es in der `tests-add.component.html` Komponente.